docs/advanced-usage/user-quotas/index.md

```diff
@@ -0,0 +1,144 @@
+In particular, if you have exceeded your CPU quota, there is nothing you can do
+except asking the administrators of your REANA instance to increase your CPU
+quota limit. However, if you have consumed all the available disk space, you can
+free up some of it by either deleting some past workflows or by removing some of
+their files.
+
+### Find workflows consuming the most disk space
+
+The easiest way to find out which workflows are using the most disk space is
+through the web interface. In particular, go to REANA's homepage and select the
+_"Most disk quota used"_ option in the _"Sort by"_ dropdown menu:
+
+![Sort workflows by "Most disk quota used"](/images/ui-sort-by-most-disk-quota-used.png)
+
+Workflows will then appear in decreasing order of disk usage: the heaviest
+workflows will be at the top of the list.
+
+Otherwise, you can do the same with `reana-client`:
+
+```console
+$ reana-client list --include-workspace-size --sort size
+NAME                      RUN_NUMBER   CREATED               STARTED               ENDED                 STATUS     SIZE
+reana-demo-root6-roofit   2            2022-05-30T12:50:46   2022-05-30T12:50:58   2022-05-30T12:51:13   finished   180224
+reana-demo-helloworld     1            2022-05-30T11:59:48   2022-05-30T12:00:00   2022-05-30T12:00:15   finished   12288
+reana-demo-root6-roofit   1            2022-05-30T11:59:58   2022-05-30T12:00:09   2022-05-30T12:01:13   deleted    0
+```
+
+### Delete workflows
+
+If there are workflows that you do not need anymore, you can delete them to free
+up some disk space. To do so, find the workflow you want to delete, click on the
+three dots on the right to open the dropwdown menu, and select _"Delete
+workflow"_:
+
+!["Delete workflow" menu option](/images/ui-delete-workflow.png)
+
+Make sure that _"Delete also workflow workspace"_ is checked before confirming
+the deletion of the workflow, otherwise the workspace will be kept intact and
+you will not free up any space:
+
+![Dialog to confirm the deletion of a workflow](/images/ui-delete-workflow-confirmation.png)
+
+You can also delete a workflow using `reana-client`, specifying the
+`--include-workspace` option to delete all the workflow's files:
+
+```console
+$ reana-client delete -w reana-demo-root6-roofit.2 --include-workspace
+==> SUCCESS: reana-demo-root6-roofit.2 has been deleted
+```
+
+See `reana-client delete --help` for more details.
+
+### Find and delete big files
+
+Once you have identified the heaviest workflows, you can also find which are the
+biggest files in their workspaces.
+
+From the web UI, open the details page of the workflow, move to the
+_"Workspace"_ tab and click on _"Size"_ to sort the files in ascending or
+descending order of size:
+
+![Sort the files in the workspace by decreasing "Size"](/images/ui-workspace-sort-by-size.png)
+
+You can also achieve the same result by using `reana-client` together with
```

Comment:
(Ditto here, we would put "Using command line, ..." CLI part before "Using web interface, ..." Web UI part.)